### ISSUE TITLE
Basics.toString leads to runtime error on Html values

### ISSUE THREAD

**Issue by berndca** [elm-lang/core#488]
I'm getting a TypeError exception for the following code:

    import Html exposing (p, text) 
    import Debug
 
    main = Debug.log "test" (p [] [ text "42"])

----------------------------------------------------------------------------------------

    09:08:01.084 TypeError: invalid 'in' operand v
	Elm.Native.Utils.make/toString() everything.js:2270
	Elm.Native.Utils.make/toString() everything.js:2364
	log() everything.js:6168
	A2() everything.js:1842
	Elm.Main.make/view() compile:24
	Elm.Main.make() compile:25
	init() everything.js:1383
	Elm.fullscreen() everything.js:1266
	<anonymous> compile:27
	1 everything.js:2270:1

**Comment by jvoigtlaender** [comment]
The error is unrelated to `Debug.log`. As evidence, it happens as well for this code:
```elm
import Html exposing (p, text) 
import Graphics.Element

main = Graphics.Element.show (p [] [ text "42"])
```
The culprit is the implementation of `Native.Utils.toString`, which both `Debug.log` and `Graphics.Element.show` are using.

**Comment by jvoigtlaender** [comment]
#183 might fix this.

**Comment by jvoigtlaender** [comment]
This is fixed in Elm 0.17.
